src/policy_evaluator.rs

```diff
@@ -16,12 +16,76 @@ use crate::cluster_context::ClusterContext;
 use crate::policy::Policy;
 use crate::validation_response::ValidationResponse;
 
+#[derive(Clone, PartialEq)]
 pub enum PolicyExecutionMode {
     KubewardenWapc,
     Opa,
     OpaGatekeeper,
 }
 
+impl serde::ser::Serialize for PolicyExecutionMode {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::ser::Serializer,
+    {
+        let mode = match self {
+            PolicyExecutionMode::KubewardenWapc => "kubewarden",
```

Comment:
We could use `"kubewarden-wapc"` and be future proof.

Reply:
Done